wafer/talks/models.py

```diff
@@ -268,3 +286,64 @@ class TalkUrl(models.Model):
     description = models.CharField(max_length=256)
     url = models.URLField()
     talk = models.ForeignKey(Talk, on_delete=models.CASCADE)
+
+
+@reversion.register(follow=('scores',))
+@python_2_unicode_compatible
+class Review(models.Model):
+    talk = models.ForeignKey(Talk, on_delete=models.CASCADE,
+                             related_name='reviews')
+    reviewer = models.ForeignKey(settings.AUTH_USER_MODEL,
+                                 on_delete=models.CASCADE)
+
+    notes = MarkupField(
+        null=True, blank=True,
+        help_text=_("Comments on the proposal (markdown)"))
+
+    def __str__(self):
+        return u'Review of %s by %s (%s)' % (
+            self.reviewer, self.talk.title, self.total_score)
+
+    @property
+    def total_score(self):
+        return self.scores.aggregate(total=models.Sum('value'))['total']
+
+    def is_current(self):
+        def last_updated(obj):
+            version = Version.objects.get_for_object(obj).first()
+            return version.revision.date_created
+        return last_updated(self) >= last_updated(self.talk)
```

Comment:
This doesn't work as expected currently, because currently revisions are created for reviews when talks are updated, because reversion's automatic registration adds all related models to the follow attribute.

I think we need to explicitly register Talks and exclude Review from being followed to have this work.

Reply:
Hrm, I didn't notice that in my testing, but the analysis sounds good.

Reply:
Fixed in a37c22b